server/controllers/books.js

```diff
@@ -17,21 +17,24 @@ const booksController = {
       author: req.body.author,
       quantity: req.body.quantity
     })
-      .then(book => res.status(201).send(book))
+      .then(() => res.status(201).json({
+        msg: 'You have a record successful'
+      }))
       .catch(() => res.status(400).json({
-        message: 'Cannot create book' }));
+        errors: [{ msg: 'Cannot create book' }]
+      }));
   },
   findBooks(req, res) {
     return db.Book.findAll({ limit: 10 })
       .then((books) => {
         if (books.length === 0) {
           return res.status(404).json({
-            message: 'No books in the library'
+            msg: 'No books in the library'
           });
         }
-        return res.status(200).json({ books });
+        return res.status(200).json(books);
       })
-      .catch(err => res.status(404).json({err}));
+      .catch(err => res.status(404).json({ err }));
```

Comment:
While you have done well in  most of your variable names. but you might want to be more detailed in certain areas. for example, it won't  hurt  to spell out `error` , `request`, `response`, etc